index-community.html

```diff
@@ -53,8 +53,14 @@
 
          <div class="inner cover">
            <h1 class="cover-heading">Documentation</h1>
-           <p class="lead"><strong>OpenShift Origin</strong> is an application platform where developers and teams can build, test, deploy, and run their applications. OpenShift Origin also serves as the upstream code base upon which <a href="http://www.openshift.com">OpenShift Online</a> and <a href="https://www.openshift.com/products/enterprise">OpenShift Enterprise</a> are built.</p>
-           <p class="lead"><a role="button" class="btn btn-lg btn-primary" href="latest/welcome/index.html"><i class="fa fa-arrow-circle-o-right"></i> Origin Latest Docs</a></p>
+           <p class="lead"><strong>OpenShift Origin</strong> is an application platform where developers and teams can build, test, deploy, and run their applications. OpenShift Origin also serves as the upstream code base upon which <a href="http://www.openshift.com">OpenShift Online</a> and <a href="https://www.openshift.com/container-platform/">OpenShift Container Platform</a> are built.</p>
+           <div class="btn-group">
+             <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Select Version<span class="caret"></span></button>
+             <ul class="dropdown-menu" role="menu">
+               <li><a href="latest/"><i class="fa fa-arrow-circle-o-right"></i> Origin Latest</a></li>
```

Comment:
That is, since there would be 1.4/3.4-related work in the master branch (somewhere between 1.3 and 1.4 currently).